src/modelInfos/fhir-modelinfo-4.0.1.xml.js

```diff
@@ -0,0 +1,12085 @@
+// This file exists so that these FHIR modelinfos can also be used in a browser where Node's 'fs' module isn't
```

Comment:
This comment applies to all the model info files

Do we need these? They're loaded into cql-exec-fhir, and we're re-using the FHIRObject class and the load function, so theoretically we shouldn't need to have the modelinfos checked into this repo at all, right?

Reply:
Good catch, I added these before confirming that we were using cql-exec-fhir as a direct dependency. I'm pretty sure we can remove them!